components/builder-api/src/server/resources/pkgs.rs

```diff
@@ -17,6 +17,7 @@ use crate::{bldr_core::{error::Error::RpcError,
             db::models::{channel::Channel,
                          origin::Origin,
                          package::{BuilderPackageIdent,
+                                   BuilderPackageSetting,
```

Comment:
Do we need a new struct or could we just leverage OriginPackageSettings?

Reply:
I actually am not 100% sure on that. Will investigate it to make sure.

Reply:
So it looks like we probably CAN do this with the existing struct, but doing so will require a bit of a refactor and we may end up returning more data on this call but that shouldn't be a deal breaker of any kind.

Reply:
Cool... I wouldn't be too worried if there's a bit more data returned on this call if we can simplify the code and concepts